src/buildstream/sandbox/_config.py

```diff
@@ -66,15 +74,15 @@ def __init__(
     # Returns:
     #    A dictionary representation of this SandboxConfig
     #
-    def to_dict(self) -> Dict[str, Union[str, int]]:
+    def to_dict(self) -> Dict[str, Union[str, int, bool]]:
 
         # Assign mandatory portions of the sandbox configuration
         #
         # /!\ No additional mandatory members can ever be added to
         #     the sandbox configuration, as that would result in
         #     breaking cache key stability.
         #
-        sandbox_dict: Dict[str, Union[str, int]] = {"build-os": self.build_os, "build-arch": self.build_arch}
+        sandbox_dict: Dict[str, Union[str, int, bool]] = {"build-os": self.build_os, "build-arch": self.build_arch}
```

Comment:
Is this outdated perhaps ? Maybe there was an added boolean here in a previous iteration of the patch ?

I think here there is only the possibility of a new string being serialized in the dict.

Reply:
Yes, there was indeed an additional boolean config option in a previous iteration. Will remove this.